feature-detects/es6/symbol.js

```diff
@@ -0,0 +1,34 @@
+/*!
+{
+  "name": "ES6 Symbol",
+  "property": "es6symbol",
+  "notes": [{
```

Comment:
maybe add a link to the mdn too 
 https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Symbol

Reply:
Added.